### PR TITLE
Fixed the bugs I found and made it responsive in Brutalist Webpage.

### PR DESCRIPTION
**Pull request by fabcodingzest**
I used some flexbox to manage the sections and made it responsive a bit.
Do you want any other changes? :)

### CODE REVIEW

File: Portfolios/Brutalist/style.css [hunk 1]
```diff
@@ -7,107 +6,118 @@ FLEXBOX
 */
 
 body {
-  margin-left: 60px;
-  margin-top: 20px;
+	margin-left: 60px;
```
Comment (jameesy) [inline Portfolios/Brutalist/style.css:9]: Hey there,

Thanks for your contribution :)

I think what has happened here is that your code formatter has changed the tab rules of the file. If you could recommit with the repo defaults, the changes displayed will be limited to just the important bits! 

Many thanks :)

**Review by jameesy — CHANGES_REQUESTED** [review]
If you could address the comment I left then that would be awesome! <3

File: Portfolios/Brutalist/style.css (hunk 1 above)
Comment (fabrizvi) [reply Portfolios/Brutalist/style.css]: Okay, I will do that.

Comment (fabcodingzest) [reply Portfolios/Brutalist/style.css:9]: Hey!
Okay so I am totally new to the PR thing but I tried to make some changes and I don't know what to do from here. I have made some more commits, could you please check and tell me if Tabs space is still a problem? I don't know how to do what you asked and made a mess ugh. I am so sorry :(

**Pull request closed without merging**